R/read_spc.R

```diff
@@ -837,19 +836,21 @@ read.spc <- function(filename,
 hySpc.testthat::test(read.spc) <- function() {
```

Comment:
I do not chack unit tests. If they pass, I'll approve them. But maybe @cbeleites or @bryanhanson can take a closer look at them?

Reply:
Yet, I recomment to do further improvements on unit test in a separate PR.

Reply:
Yes, hold for separate PR.

Reply:
I created a separate issue for this: #7